### ISSUE TITLE
HIP DRAFT: Reduce the cost of Helium IoT Network

### ISSUE THREAD

**Issue by oosui** [helium/HIP#483]
# Why?

Almost 1 million hotspots have been deployed but few use. One of the reason is the cost is too high.

It seems the cost of building an IoT solution is $1/year per sensor which is a low price, right?
But actually it's not cheap at all if you start to build an application on top of it.

Suppose you have 100k sensors which is a small number, how much will you spend every year?

### Cost

```
> $120,000
```

### Hotspots

Since hotspots are hosted and managed by anyone, you can't trust them because they can shutdown hotspots at any time.
It will be a big risk to use the network in such situation. So you have to have your own hotspots deployed to make sure your 
application works `24*7*365`.

In that case, suppose you need to deploy 1 hotspot every 1000 sensors, the cost of hotspots will be 
`$40` (onboarding) + `$20` (assert location 2 times average) + `$?` (hardware cost).

So the basic cost of hotspots will be:

```
100,000 / 1000 * ($40 + $20 + >$140) > $20,000
```

or data only hotspots:

```
100,000 / 1000 * ($10 + $5 + >$140) > $16,000
```

### Sensors

Suppose the cost of packet transfer is about $1 per year.
Onboarding sensors cost more.

For a sensor with uplink && ack every 10 minutes,
DC cost:
```
6 * 24 * 365 * 2 = 105,120
```

DC has a fixed USD cost of `$0.00001`. 
So the total cost of a sensor will be `$1.0512`

So for 100,000 sensors, the cost will be:
```
100,000 * $1 = $100,000
```

### Softwares

Software cost not included such as Console, Router, ETL, etc.
Basically all applications will need Console, Router, ETL, etc since 
there isn't an existing one for users to use. Not sure how much it will take
let's say $10000 / year? Is it OK for 100k devices?

### Cost

```
$64,436
```

### Hotspots

The basic cost of hotspots will be:

```
100,000 / 1000 * (>$140) > $14,000
```

* No onboarding / location assertion fee.
* No ECC chip, etc. Price should be less than the number above.

### Sensors

All the cost included in the plan.
https://accounts.thethingsindustries.com/fee-calculator

```
$50,436 / year
```

### Softwares

All the cost included in the plan.

### Reduce the light hotspot onboarding && location assertion fee.

`$20` / light gateway

`$5`  / location assertion

Reduce DC for data transfer.

Helium:

```
Cost
|                       .
|                  .
|             .
|        .
|______________________________________________________________
Sensors

```

TTN:

```
Cost
|                                                  .
|                  .
|      .           
|        
|_._____________________________________________________________
Sensors

```

**Comment by kellybyrd** [comment]
> Since hotspots are hosted and managed by anyone, you can't trust them because they can shutdown hotspots at any time.
It will be a big risk to use the network in such situation. So you have to have your own hotspots deployed to make sure your
application works 24*7*365.

I don't understand your logic.  If this is true, the whole concept of the Helium network fails. The whole point is so users of the network don't have to deploy hotspots. If they do, it would be easier for an IoT deployment to not use Helium, and instead do a standard LoRa deployment and not pay for the network usage.

**Comment by jeanmouawad** [comment]
Helium is Already extremely cheap. 
- You are paying 
Approx 5$ 
For using 1 sensor transmitting data every 1 minute. 
- You are comparing to TTN which it doesn't cover 20% of Helium Coverage. 
- How are you planning to incentivize people for data transfer if you want to reduce the cost of usage ? 
- In addition you don't need 10 000$ for IoT management software since using this amount of money can help you develop a full IoT Management Platform with cloud database ( since data transfer packets are low in size ) and resell this Platform. 
- Onboarding hotspots is paid by companies that manufacture these gateways not the user. And asserting location twice a year is not logical too much 
- Finally how is 100k sensors a small number ?

**Comment by oosui** [comment]
> > Since hotspots are hosted and managed by anyone, you can't trust them because they can shutdown hotspots at any time.
> > It will be a big risk to use the network in such situation. So you have to have your own hotspots deployed to make sure your
> > application works 24_7_365.
> 
> I don't understand your logic. If this is true, the whole concept of the Helium network fails. The whole point is so users of the network don't have to deploy hotspots. If they do, it would be easier for an IoT deployment to not use Helium, and instead do a standard LoRa deployment and not pay for the network usage.

If you start to use the network you will notice that some hotspots are not having real coverage. For applications are not critical such as temperature, humidity, nothing to worry about. But for some special use cases such as emergency push button, fire alarm, water leak sensor, it will be critical to control the gateway and know the gateway status to make sure these sensors are covered. When selling these products, you don't know where they will be deployed but in most of the cases, they will be used in basement, on a hill, etc. A good way is to deploy a gateway together with these devices. Yes, you are right, maybe for such kind of use cases, Helium is not the best choice.

**Comment by oosui** [comment]
> Helium is Already extremely cheap.
> 
> * You are paying
>   Approx 5$
>   For using 1 sensor transmitting data every 1 minute.
> * You are comparing to TTN which it doesn't cover 20% of Helium Coverage.
> * How are you planning to incentivize people for data transfer if you want to reduce the cost of usage ?
> * In addition you don't need 10 000$ for IoT management software since using this amount of money can help you develop a full IoT Management Platform with cloud database ( since data transfer packets are low in size ) and resell this Platform.
> * Onboarding hotspots is paid by companies that manufacture these gateways not the user. And asserting location twice a year is not logical too much
> * Finally how is 100k sensors a small number ?

Thanks for your reply! 
Yes, it's cheap since it has good coverage. Best for a tracker.
But with the increase of sensors, the data transfer cost is expensive.

For example you build an IoT Management Platform to support `100,000` sensors with `$5/sensor/year`.

So for TTN, the data transfer cost is fixed, `$50,436` / year.
But on Helium, since it's charged by device, not the service, the cost will be `$500,000`, right?

If you have `1,000,000` sensors?  The cost will be `$5,000,000`. But for TTN, it will be `<$500,000`. By the way, this is the data transfer cost only.

100k is a small number compare to the real sensor market size.

For the onboarding fee, it's the case that a service provider has to know the real time status of their gateways for some critical use cases.

**Comment by TheRealJohnMac50** [comment]
> # Why?
> Almost 1 million hotspots have been deployed but few use. One of the reason is the cost is too high.
> 
> It seems the cost of building an IoT solution is $1/year per sensor which is a low price, right? But actually it's not cheap at all if you start to build an application on top of it.
> 
> Suppose you have 100k sensors which is a small number, how much will you spend every year?
> 
> ## Basic Cost Use Helium IoT Network
> ### Cost
> ```
> > $120,000
> ```
> 
> ### Hotspots
> Since hotspots are hosted and managed by anyone, you can't trust them because they can shutdown hotspots at any time. It will be a big risk to use the network in such situation. So you have to have your own hotspots deployed to make sure your application works `24*7*365`.
> 
> In that case, suppose you need to deploy 1 hotspot every 1000 sensors, the cost of hotspots will be `$40` (onboarding) + `$20` (assert location 2 times average) + `$?` (hardware cost).
> 
> So the basic cost of hotspots will be:
> 
> ```
> 100,000 / 1000 * ($40 + $20 + >$140) > $20,000
> ```
> 
> or data only hotspots:
> 
> ```
> 100,000 / 1000 * ($10 + $5 + >$140) > $16,000
> ```
> 
> ### Sensors
> Suppose the cost of packet transfer is about $1 per year. Onboarding sensors cost more.
> 
> For a sensor with uplink && ack every 10 minutes, DC cost:
> 
> ```
> 6 * 24 * 365 * 2 = 105,120
> ```
> 
> DC has a fixed USD cost of `$0.00001`. So the total cost of a sensor will be `$1.0512`
> 
> So for 100,000 sensors, the cost will be:
> 
> ```
> 100,000 * $1 = $100,000
> ```
> 
> ### Softwares
> Software cost not included such as Console, Router, ETL, etc. Basically all applications will need Console, Router, ETL, etc since there isn't an existing one for users to use. Not sure how much it will take let's say $10000 / year? Is it OK for 100k devices?
> 
> ## Basic Cost Use TTN
> ### Cost
> ```
> $64,436
> ```
> 
> ### Hotspots
> The basic cost of hotspots will be:
> 
> ```
> 100,000 / 1000 * (>$140) > $14,000
> ```
> 
> * No onboarding / location assertion fee.
> * No ECC chip, etc. Price should be less than the number above.
> 
> ### Sensors
> All the cost included in the plan. https://accounts.thethingsindustries.com/fee-calculator
> 
> ```
> $50,436 / year
> ```
> 
> ### Softwares
> All the cost included in the plan.
> 
> ## How to reduce the cost?
> ### Reduce the light hotspot onboarding && location assertion fee.
> `$20` / light gateway
> 
> `$5` / location assertion
> 
> Reduce DC for data transfer.
> 
> Helium:
> 
> ```
> Cost
> |                       .
> |                  .
> |             .
> |        .
> |______________________________________________________________
> Sensors
> ```
> 
> TTN:
> 
> ```
> Cost
> |                                                  .
> |                  .
> |      .           
> |        
> |_._____________________________________________________________
> Sensors
> ```

Covered at least some of this in HIP 69:

https://github.com/helium/HIP/blob/main/0069-reassertion-fee-reduction.md

**Comment by vincenzospaghetti** [comment]
This issue was linked to a previously closed PR #482 https://github.com/helium/HIP/pull/482.

**Comment by Aminkavoos** [comment]
Hello
